fearless/Modules/NewWallet/WalletSend/WalletSendViewController.swift

```diff
@@ -52,21 +52,24 @@ final class WalletSendViewController: UIViewController, ViewHolder {
     }
 
     private func applyState(_ state: WalletSendViewState) {
+        self.state = state
+
         switch state {
         case .loading:
             break
-        case let .loaded(walletSendViewModel):
-            if let accountViewModel = walletSendViewModel.accountViewModel {
+        case let .loaded(model):
+            if let accountViewModel = model.accountViewModel {
                 rootView.bind(accountViewModel: accountViewModel)
             }
 
-            rootView.bind(feeViewModel: walletSendViewModel.feeViewModel)
+            rootView.bind(feeViewModel: model.feeViewModel)
+            rootView.bind(tipViewModel: model.tipViewModel, isRequired: model.tipRequired)
```

Comment:
May be just hide tip view by default and show it if model isn't empty? Also if tip isn't required may be we shouldn't bind this model at all.